BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -20,7 +20,7 @@
 "ginibank.help.menu.returnAssistant.section3.body" = "Sollte der Preis mal nicht perfekt ausgelesen worden sein, können Sie diesen einfach manuell korrigieren. Der Gesamtpreis aktualisiert sich automatisch.";
 "ginibank.help.menu.returnAssistant.backButton.title" = "Zurück";
 
-"ginibank.digitalinvoice.screentitle" = "Digitale Rechnung";
+"ginibank.digitalinvoice.screentitle" = "Digitale rechnung";
```

Comment:
[Request] `Rechnung` has to remain capital case due to the German grammar rules (Germans write all nouns in capital case).